3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/registry/RefererParserEnrichment.scala

```diff
@@ -49,7 +49,7 @@ object RefererParserEnrichment extends ParseableEnrichment {
 
   implicit val formats = DefaultFormats
 
-  val supportedSchema = SchemaCriterion("com.snowplowanalytics.snowplow", "referer_parser", "jsonschema", 1, 0)
+  val supportedSchema = SchemaCriterion("com.globeandmail", "sophi_referer_parser", "jsonschema", 1, 0)
```

Comment:
why is this a new version? can we increate the version of the snowplow one? or we do `"com.globeandmail", "referer_parser", "jsonschema", 2, 0`?

Reply:
No, because it's not actually the snowplow one. It's our own modified schema, which Snowplow is not going to use. If they ever updated it themselves, it would cause a version collision.

Reply:
then please remove `sophi_` from it.

Reply:
Done.